docs/deployment/on-chain_deployment_checklist.md

```diff
@@ -179,18 +178,38 @@ See also the [service level overrides](../guides/service_configuration_file.md#s
 
       params:
         args:
-          consensus:
-            max_participants:  ${MAX_PARTICIPANTS:int:4}
           setup:
             all_participants: ${ALL_PARTICIPANTS:list:['["0x...","0x...","0x...","0x..."]']}      
             safe_contract_address: ${SAFE_CONTRACT_ADDRESS:list:[0x...]}
+            consensus_threshold: ${CONSENSUS_THRESHOLD:int:null}
           tendermint_url: ${TENDERMINT_URL:str:http://localhost:26657}
           tendermint_com_url: ${TENDERMINT_COM_URL:str:http://localhost:8080}
           service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:0x...}
           share_tm_config_on_startup: ${SHARE_TM_CONFIG_ON_STARTUP:bool:false}
           on_chain_service_id: ${ON_CHAIN_SERVICE_ID:int:1}
           (...)
     ```
+!!! warning "Important"
+    Recall that when [deploying an on-chain service](../guides/deploy_service.md#on-chain-deployment) using `autonomy deploy from-token`, a number of arguments are overridden with the values registered in the Autonolas Protocol:
+    ```yaml
+    (...)
+    models:
+      params:
+        args:
+          setup:
+            all_participants: # Overridden with the registered values
+            safe_contract_address: # Overridden with the registered values
+            consensus_threshold: # Overridden with the registered values
+          service_registry_address:  # Overridden with the registered values
+          on_chain_service_id: # Overridden with the registered values
```

Comment:
No parameters outside of the `setup` config will be overridden when running the `from-token` command